### PR TITLE
Node 18

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What type of PR is this? 
Node 18 is not the latest stable, but at least still maintained in contrast to 16 and 14. Tested it with Node 20 as well, but currently there's no cypress image with node 20

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

### CODE REVIEW

**Comment by AndrewChubatiuk** [comment]
@justinclift @guidopetri could you please review it?

**Comment by justinclift** [comment]
@AndrewChubatiuk I'm taking some time off after hitting the early stages of burn out. :frowning:

File: package.json [hunk 1]
```diff
@@ -7,14 +7,13 @@
...
-    "watch:app": "webpack watch --progress",
+    "build": "yarn clean && yarn build:viz && NODE_OPTIONS=--openssl-legacy-provider NODE_ENV=production webpack",
+    "watch:app": "NODE_OPTIONS=--openssl-legacy-provider webpack watch --progress",
     "watch:viz": "(cd viz-lib && yarn watch:babel)",
     "watch": "npm-run-all --parallel watch:*",
     "webpack-dev-server": "webpack-dev-server",
-    "analyze": "yarn clean && BUNDLE_ANALYZER=on webpack",
-    "analyze:build": "yarn clean && NODE_ENV=production BUNDLE_ANALYZER=on webpack",
+    "analyze": "yarn clean && BUNDLE_ANALYZER=on NODE_OPTIONS=--openssl-legacy-provider webpack",
+    "analyze:build": "yarn clean && NODE_ENV=production BUNDLE_ANALYZER=on NODE_OPTIONS=--openssl-legacy-provider webpack",
```
Comment (justinclift) [inline package.json:16]: Sounds like this is needed for NodeJS 18 to work with the Redash repo then yeah?

Comment (AndrewChubatiuk) [reply package.json:16]: this is needed to work with machines with old version of openssl

Comment (justinclift) [reply package.json:16]: Cool, no worries. :smile:

**Review by justinclift — APPROVED** [review]
Looks good to me. :smile:

**Comment by AndrewChubatiuk** [comment]
@justinclift wanted to merge #6674

**Comment by justinclift** [comment]
@AndrewChubatiuk Interesting.  That looks like it'll need some proper thinking about to fix, which I don't have the mental bandwidth for at the moment. :frowning:

Saying that because I'm under the impression it shouldn't be running scenarios from `master` when the PR is in a different branch.

If that's what's really happening, then that sounds like a problem in itself. :frowning:

**Comment by eradman** [comment]
@AndrewChubatiuk can you update https://github.com/getredash/redash/wiki/Local-development-setup to match?  I assume this change implies that we should be running `nvm use 18` for development now.

**Comment by snickerjp** [comment]
I have made an UPDATE to the wiki.
https://github.com/getredash/redash/wiki/Local-development-setup/_compare/a534424f2d8da9221e29019ed12bef94c6d4ed2f...c5c70019e6e0ae773b9a35f6d5a722ccde710616

**Comment by justinclift** [comment]
Awesome.  Thanks heaps @snickerjp. :smile:

**Pull request merged**